editor/components/rich-text/index.js

```diff
@@ -829,7 +829,6 @@ export class RichText extends Component {
 			formattingControls,
 			placeholder,
 			multiline: MultilineTag,
-			keepPlaceholderOnFocus = false,
```

Comment:
It should be also removed from Readme file: https://github.com/WordPress/gutenberg/blob/4af0d34480a22623d4c03a436c2bd7628bc710ba/editor/components/rich-text/README.md#keepplaceholderonfocus-boolean